### ISSUE TITLE
Security vulnerabilities in 1.21.3-alpine

### ISSUE THREAD

**Issue by KirylBubovich** [nginxinc/docker-nginx#577]
I'm getting the following `curl` vulnerabilities 
```
+------------+------------------------+--------------+-----------------+---------------------------------------------------------------+
| Unapproved | Unknown CVE-2021-22945 | curl         | 7.78.0-r0       |                                                               |
|            |                        |              |                 | https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2021-22945 |
+------------+------------------------+--------------+-----------------+---------------------------------------------------------------+
| Unapproved | Unknown CVE-2021-22947 | curl         | 7.78.0-r0       |                                                               |
|            |                        |              |                 | https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2021-22947 |
+------------+------------------------+--------------+-----------------+---------------------------------------------------------------+
| Unapproved | Unknown CVE-2021-22946 | curl         | 7.78.0-r0       |                                                               |
|            |                        |              |                 | https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2021-22946 |
+------------+------------------------+--------------+-----------------+---------------------------------------------------------------+
```
This temporary solution is currently being used to avoid vulnerabilities
`RUN apk add --update --no-cache 'curl>=7.79.0-r0'`

**Comment by joakimnordling** [comment]
`1.20.1-alpine` also ships with the same vulnerable version of `curl` and `libcurl`(`7.78.0-r0`).

**Comment by thresheek** [comment]
Fixed in `nginx:1.21.4-alpine`.
